src/ast/constants.ts

```diff
@@ -89,6 +89,32 @@ export enum TimeUnit {
     Years = "years"
 }
 
+export enum CompilationOutput {
```

Comment:
Would be cool to have Solidity reference documentation entry for this enum. Also, this file is for AST-related constants. Compile-related constants better to be stored in `src/compile/constants.ts`.

Reply:
Fixed in 125b9f3